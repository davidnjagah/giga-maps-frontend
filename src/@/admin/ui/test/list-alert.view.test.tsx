import { render } from "@testing-library/react"

import { testWrapper } from "~/tests/jest-wrapper"

import ListAlertView from "../alerts/list-alert.view"

describe("ListAlertView", () => {
  test("Render ListAlertView", () => {
    const { asFragment } = render(testWrapper(<ListAlertView />))
    expect(asFragment()).toMatchSnapshot();
  })
})
