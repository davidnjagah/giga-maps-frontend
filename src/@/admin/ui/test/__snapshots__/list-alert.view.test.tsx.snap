// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListAlertView Render ListAlertView 1`] = `
<DocumentFragment>
  .c1 >h3 {
  color: #FFF;
  font-family: Open Sans;
  font-size: 1.5rem;
  font-style: normal;
  font-weight: 500;
  line-height: 1.75rem;
}

.c1 >p {
  color: #F5F5F5;
  font-family: Open Sans;
  font-size: 0.875rem;
  font-style: normal;
  font-weight: 400;
  line-height: 1.125rem;
}

.c0 {
  padding: 3.75rem 2rem 1rem 1rem;
  background: #222;
  display: flex;
  align-items: center;
  justify-content: space-between;
}

<div
    class="c0"
  >
    <div
      class="c1"
    >
      <h3>
        Alerts
      </h3>
      <p>
        Notification about changes and updates
      </p>
    </div>
  </div>
</DocumentFragment>
`;
